setup.py

```diff
@@ -58,7 +58,7 @@ def read(fname):
       url="https://github.com/ome/omero-signup",
       download_url='https://github.com/ome/omero-signup/archive/v%s.tar.gz' % VERSION,  # NOQA
       keywords=['OMERO.web', 'plugin'],
-      install_requires=['omero-web>=5.6.0'],
+      install_requires=['omero-web>=5.6.0,<5.14.0'],
```

Comment:
Shouldn't this be `<6.0.0` in case of future patches?

Reply:
Reading https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api
It seems that we are doing what is recommended